libraries/abstractions/secure_sockets/utest/secure_sockets_utest.c

```diff
@@ -234,6 +255,29 @@ void test_SecureSockets_send_successful( void )
     deinitSocket( so );
 }
 
+/*!
+ * @brief A happy send case after bind with normal sockets
+ *
+ * @details The purpose is to make sure we get returned the same number of bytes
+ *          That lwip_send returned
+ */
+void test_SecureSockets_send_successful_after_bind ( void )
+{
+	int32_t ret;
+	Socket_t so = create_connection_after_bind ();
+
+	const char buffer[ BUFFER_LEN ];
+
+	lwip_send_ExpectAnyArgsAndReturn( BUFFER_LEN );
+	ret = SOCKETS_Send( so,
+	                        buffer,
+	                        BUFFER_LEN,
+	                        0 );
```

Comment:
```suggestion
	ret = SOCKETS_Send( so,
	                                   buffer,
	                                   BUFFER_LEN,
	                                   0 );
```

Reply:
Hi @aggarw13 ,

I have an email from Amazon (forwarded by my manager) to remove the unit test changes from this PR and file a separate PR
Thanks
Balaji.

Reply:
Removed unit test from the PR.

Reply:
Yes, that is fine to incorporate the unit tests in a separate PR.